api_utils/except_err.py

```
@@ -0,0 +1,60 @@
+
+from .except_err import exception_err
+from .except_err import exception_err, warn_err
+
+"""
+import inspect
+import traceback
+from warnings import warn
+
+try:
+    from . import printe
+except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)